ghcide/ghcide.cabal

```diff
@@ -45,77 +43,76 @@ common warnings
 
 library
   import: warnings
-  default-language:   GHC2021
+  default-language: GHC2021
   build-depends:
-    , aeson
-    , array
-    , async
-    , base                         >=4.16     && <5
-    , base16-bytestring            >=0.1.1    && <1.1
-    , binary
-    , bytestring
-    , case-insensitive
-    , co-log-core
-    , containers
-    , cryptohash-sha1              >=0.11.100 && <0.12
-    , data-default
-    , deepseq
-    , dependent-map
-    , dependent-sum
-    , Diff                         ^>=0.4.0
-    , directory
-    , dlist
-    , enummapset
-    , exceptions
-    , extra                        >=1.7.14
-    , filepath
-    , fingertree
-    , focus                        >=1.0.3.2
-    , ghc                          >=9.2
-    , ghc-boot
-    , ghc-boot-th
-    , ghc-check                    >=0.5.0.8
-    , ghc-paths
-    , ghc-trace-events
-    , Glob
-    , haddock-library              >=1.8      && <1.12
-    , hashable
-    , hie-bios                     ^>=0.14.0
-    , hie-compat                   ^>=0.3.0.0
-    , hiedb                        ^>= 0.6.0.0
-    , hls-graph                    == 2.8.0.0
-    , hls-plugin-api               == 2.8.0.0
-    , implicit-hie                 >= 0.1.4.0 && < 0.1.5
-    , lens
-    , list-t
-    , lsp                          ^>=2.5.0.0
-    , lsp-types                    ^>=2.2.0.0
-    , mtl
-    , opentelemetry                >=0.6.1
-    , optparse-applicative
-    , parallel
-    , prettyprinter                >=1.7
-    , prettyprinter-ansi-terminal
-    , random
-    , regex-tdfa                   >=1.3.1.0
-    , safe-exceptions
-    , sorted-list
-    , sqlite-simple
-    , stm
-    , stm-containers
-    , syb
-    , text
-    , text-rope
-    , time
-    , transformers
-    , unliftio                     >=0.2.6
-    , unliftio-core
-    , unordered-containers         >=0.2.10.0
-    , vector
+    Diff ^>=0.4.0,
+    Glob,
```

Comment:
I chose to make Gild sort things case sensitively, but I'm not set on it. See https://github.com/tfausak/cabal-gild/issues/46.